internal/builders/docker/common.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2022 SLSA Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
I would move the BuildConfig (which I think is the input configuration) into a sub-package instead of main, something like `internal/builders/docker/pkg/common.go`

Reply:
That is a good idea. Thanks. What would you call that package? `common`? (I am a bit confused by the folder/package structure).

Reply:
That package would just be in `package pkg`, but if you wanted to sub-dir it I suppose you could -- although I don't think there will be many more files